csrankings.csv

```diff
@@ -15884,3 +15884,32 @@ Peter I. Cowling,University of York,https://www-users.cs.york.ac.uk/~pic/,xQ7IpL
 Delaram Kahrobaei,University of York,https://www-users.cs.york.ac.uk/~delaram/,ScTS9PkAAAAJ
 Jim Austin,University of York,https://www-users.cs.york.ac.uk/~austin/,xpZL8d8AAAAJ
 James Alfred Walker,University of York,https://www.cs.york.ac.uk/people/?group=Research%20Fellows%20and%20Research%20Associates&username=jawalker,Yl5OycsAAAAJ
+Rodrigo Agerri,University of the Basque Country UPV/EHU,www.rodrigoagerri.net,1SJh0a8AAAAJ

```

Comment:
According to the commit message, this is only adding a specific group. You need to add the entire faculty if adding a new institution to the database. Also, all of these people need to be professors; this seems like a lot of people just in NLP.

Reply:
Thanks for the quick reply.
I'vw solved the space issue in the aliases file.
Also, I've removed non tenure-track professors from the list (sorry for including those, I misread the requirements).